### DISCUSSION TITLE
Could someone tell me how to manually install the Nitpick patch for 7 with the Keyring mod?

### DISCUSSION THREAD

**Discussion by SeanW49** [exult/exult#538] [General]
All the files I would need to copy, and to where. I read a discussion in  2023  by Zhelkas, #374, that it was possible and it mentioned the USECODE file. Do I just replace the one file or others as well? I tried to run the installer in Windows and telling it my installation directory was "D:\Exult\blackgate", that didn't work. I tried copying it to C:\, that didn't help. Any help will be appreciated. Thank you in advance.

**Comment by GrantWChapman** [comment]
I'll give it a shot after work tomorrow and let you know.

**Comment by SeanW49** [comment]
Thank you good sir, whenever you have the time.

**Reply by GrantWChapman** [reply]
Okay. It looks like it's actually pretty simple to get it to work, at least with the minimal testing I've done. (I've tested by comparing Gwenno's scroll.)
Here's a comparison screenshot to show that it works.
![U7ModCompare](https://github.com/user-attachments/assets/731dc426-83dc-479b-8a5e-262461a741ff)

Here's the way I installed the Nitpicker patch to get it to work with the Keyring mod.
1. Back up your original Ultima7 directory.
2. Extract the nitpicker patch to a directory.
3. Run the batch installer and point to the Ultima 7 directory. (In my case it is D:\Programs\Origin Systems\ULTIMA7).
4. Press [1] Do you play the original DOSBOX version.
5. Press any key to install.
6. Press [1] US-AMERICAN layout.
7. Press [1] original Soundblaster.
This will install the mod to run on the unmodified Ultima7 game, however it also appears to work in Exult, however it does not appear as a separate mod just the regular game. You can then extract the keyring mod into the mods folder and it should work like the attached image.

**Comment by SeanW49** [comment]
Okay, I'll see if it works.  Ty for the help!
